packages/ui/package.json

```diff
@@ -33,9 +33,13 @@
     "build:storybook": "storybook build -c .storybook -o build/",
     "predeploy": "npm run build:storybook",
     "deploy": "gh-pages -d build/",
-    "storybook": "storybook dev -p 6006 -h travis.shetland-turtle.ts.net --disable-telemetry --https --ssl-key ../../certs/key.pem --ssl-cert ../../certs/cert.pem"
+    "storybook": "storybook dev -p 6006 -h tcb-mbp.shetland-turtle.ts.net --disable-telemetry --https --ssl-key ../../certs/key.pem --ssl-cert ../../certs/cert.pem"
```

Comment:
This has your personal machine address in it, probably should change this.

Reply:
Oh yeah. I haven't figured out how to pull that from the .env.local file yet. That update is coming in my storybook branch.

Reply:
I'll take it out for now.